internal/controllers/admission_test.go

```diff
@@ -61,6 +61,14 @@ var _ = Describe("Operator Spec Validations", func() {
 			"1.2.3-pre+bad_metadata",
 			"1.2.-3",
 			".1.2.3",
+			"<<1.2.3",
+			">>1.2.3",
+			">~1.2.3",
+			"==1.2.3",
+			"=!1.2.3",
+			"1.Y",
+			">1.2.3 && <2.3.4",
+			">1.2.3;<2.3.4",
```

Comment:
Should we add some valid masterminds range constraints to _our_ invalid list? (e.g. `"1.2.3-2.0.0"`)

Reply:
I was _just_ thinking about the hyphen range!
So, probably yes.